Feat(eos_cli_config_gen)!: Add support to set interval for each recovery cause in errdisable (#5496)
aristanetworks/ansible-avd: docs/porting-guides/6.x.x.md

+    vrf: default
```

> ### `errdisable.recovery.causes` is now a dictionary

Reviewer: ```suggestion
### `errdisable.recovery.causes` is now a list of dictionaries
```

In previous versions of AVD, `errdisable.recovery.causes` was a list of strings.
Starting AVD 6.0, this model is improved by supporting configuration of recovery interval for each cause. `errdisable.recovery.causes` is a now a list of dictionaries. Refer below diff to update the inputs according to new data model.

```diff
errdisable:
  recovery:
-   causes: [bgpguard, tapagg]
+   causes:
+     - name: bgpguard

> +     - name: tapagg

Reviewer: ```suggestion
-   causes: [bpduguard, tapagg]
+   causes:
+     - name: bpduguard
+     - name: tapagg
```

+    vrf: default
```

### `errdisable.recovery.causes` is now a list of dictionaries

In previous versions of AVD, `errdisable.recovery.causes` was a list of strings.

> Starting AVD 6.0, this model is improved by supporting configuration of recovery interval for each cause. `errdisable.recovery.causes` is a now a list of dictionaries. Refer below diff to update the inputs according to new data model.

Reviewer 2: ```suggestion
In AVD 5.x, `errdisable.recovery.causes` was modeled as a list of strings.

As of AVD 6.0.0, this data model has been updated to a list of dictionaries to allow configuring a recovery interval for each cause.

The diff below shows an example on how to update your inputs to match the new data model:
```
